src/Control/Monad/STE.hs

```diff
@@ -142,8 +122,8 @@ privateCatch (STE steAction)  = ST $ unsafeCoerceIOAction2StateS $
 --                       Nothing -> raiseIO# e
 
 
-{-# INLINE  handleSTE #-}
```

Comment:
we already discussed this, but lets try to have formatting cleanup stuff (in the future) be in their own commits. (doesn't matter for now, but worth mentioning for future contributors)